src/pages/Integrations/Create/schema.ts

```diff
@@ -15,33 +15,96 @@ export const schema = (category) => ({
         'Configure integrations between third-party tools and the Red Hat Hybrid Cloud Console.',
       name: 'add-integration-wizard',
       fields: [
-        integrationsStep(category),
+        ...(['Communications'].includes(category)
```

Comment:
@aferd FYI you can add your category in here.

@radekkaluzik pleace change this to a constant: `IntegrationCategory.COMMUNICATIONS` imported from `src/types/Integration.ts`

Reply:
Fixed